src/common/utils/api-request.ts

```diff
@@ -0,0 +1,52 @@
+import axios, { AxiosResponse } from 'axios';
+
+const apiBaseUrl: string =
+  process.env.REACT_APP_API_BASE_URL || 'http://localhost:8000';
```

Comment:
Currently port number is 3000, so maybe change the default 8000 -> 3000 here.

Reply:
This is actually the address of local Hauki-api

Reply:
True, it is correct now